src/style.css

```
@@ -0,0 +1,84 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 900;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+.img {
+  position: absolute;
+  top: 25px;
+  left: 26px;
+}
+
+.img--small {
+  top: 15px;
+  left: 13px;
+  width: 11px;
+  height: 11px;
+}
+
+.form {
+  margin: 20px 0;
+  position: relative;
+}
+
+.input {
+  background: #fff;
+  outline: none;
+  width: 100%;
+  font-family: Avenir, sans-serif;
```

move it to html tag selector and let input inherit it 
```suggestion
  font-family: inherit;
```